homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

```diff
@@ -0,0 +1,177 @@
+"""Support for Atlantic Pass APC DHW."""
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.water_heater import (
+    STATE_ECO,
+    STATE_OFF,
+    STATE_PERFORMANCE,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from .. import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_OPERATION_MODE: dict[str, str] = {
+    STATE_PERFORMANCE: OverkizCommandParam.PERFORMANCE,
+    STATE_ECO: OverkizCommandParam.MANUAL_ECO_ACTIVE,
+    OverkizCommandParam.MANUAL: OverkizCommandParam.MANUAL_ECO_INACTIVE,
+}
```

Comment:
I am not sure if I understand this dict. Should'nt the left side only have OverkizCommandParam values and the right side only valid 'operation_modes'?

Reply:
And you are not using this dict anywhere, except for the mapping to operation list.